execution/aggregate/khashaggregate.go

```diff
@@ -138,6 +142,23 @@ func (a *kAggregate) GetPool() *model.VectorPool {
 	return a.vectorPool
 }
 
+func (a *kAggregate) Analyze() (model.OperatorTelemetry, []model.ObservableVectorOperator) {
+	if _, ok := a.OperatorTelemetry.(*model.TimingInformation); ok {
```

Comment:
Yep, we don't need these. I think Nischay just forgot to remove.

Reply:
Yeah I have to remove it at some places, but I think we need to type check to check if it is of the type `TimingInformation` and not `NoopTimingInformation` to return it in Analyze.

Reply:
Why? `kAggregate` already implements `model.OperatorTelemetry` so we can return it directly, no?

Reply:
Type checks removed